rules/order.js

```diff
@@ -30,6 +55,7 @@ module.exports = {
         'box-sizing',
         'display',
         'overflow',
+        'vertical-align',
```

Comment:
This sort of stuff gives me the heebie jeebies about this whole thing. The order is often so, so subjective, and there are hundreds of properties. It's an unreasonable ask, IMO, that _everyone_ actually get on board with an at least somewhat arbitrary grouping when there is absolutely not necessarily one 'right way' form a logical standpoint. This will require back and forth with the linter in one way or another for the majority of contributors, there is no way around it in my view. That is what we lose.